geo/src/algorithm/area.rs

```diff
@@ -186,8 +186,8 @@ where
 }
 
 /// **Note.** The implementation is a straight-forward
-/// summation of the signed areas of the individual
-/// polygons. In particular, `unsigned_area` is not
+/// summation of the signed or unsigned areas of the individual
```

Comment:
@rmanoka, you added this note in eb693154ee488639e3c46b73d258c16fdbefc8c6. Am I misunderstading something?